kubectl-pg/glide.yaml

```diff
@@ -0,0 +1,21 @@
+package: github.com/VineethReddy02/postgres-operator/kubectl-pg
+import:
```

Comment:
please consider pinning versions of packages where it makes sense (i.e. specifying the [version as in the operators' `glide.yaml`](https://github.com/zalando/postgres-operator/blob/master/glide.yaml#L3)). This can potentially save you a lot of headache later when the dependencies evolve in unpredictable ways